src/query/ast/src/parser/statement.rs

```diff
@@ -895,16 +895,19 @@ pub fn statement(i: Input) -> IResult<StatementMsg> {
         rule! {
             PRESIGN ~ ( #presign_action )?
                 ~ #presign_location
-                ~ (EXPIRE ~ "=" ~ #literal_u64)?
+                ~ ( #presign_option )*
         },
-        |(_, action, location, expire)| {
-            Statement::Presign(PresignStmt {
+        |(_, action, location, opts)| {
+            let mut presign_stmt = PresignStmt {
                 action: action.unwrap_or_default(),
                 location,
-                expire: expire
-                    .map(|(_, _, v)| Duration::from_secs(v))
-                    .unwrap_or_else(|| Duration::from_secs(3600)),
-            })
+                expire: Duration::from_secs(3600),
+                content_type: None,
```

Comment:
How about use `application/octet-stream` as default value if not input?

Reply:
makes sense, but i'm a bit afraid of this may break the existing code in SDKs, if it defaults as None, a request without Content-Type header can be successful, however if we make the default value as application/octet-stream, the code without setting Content-Type to application/octet-stream would fail :(

Reply:
Based on our `presign` API design, it's required to add all headers that we returned into requests to make it work.

Reply:
mark the conversation as resolved?

Reply:
after talk, keeping the behavior with current is ok